tests/functional/lib/helpers.js

```diff
@@ -679,44 +680,45 @@ define([
    * @param {boolean} [options.untrusted] - if `true`, opens the Untrusted
    * relier. Defaults to `true`
    */
-  function openFxaFromRp(context, page, options) {
+  function openFxaFromRp(page, options) {
     options = options || {};
     var app = options.untrusted ? UNTRUSTED_OAUTH_APP : OAUTH_APP;
     var expectedHeader = options.header || '#fxa-' + page.replace('_', '-') + '-header';
     var queryParams = options.query || {};
 
-    // force_auth does not have a button on 123done, instead this is
-    // only available programatically. Load the force_auth page
-    // with only the email initially, then reload with the full passed
-    // in urlSuffix so things like the webChannelId are correctly passed.
+    return function () {
+      // force_auth does not have a button on 123done, instead this is
+      // only available programatically. Load the force_auth page
+      // with only the email initially, then reload with the full passed
+      // in urlSuffix so things like the webChannelId are correctly passed.
+      if (page === 'force_auth') {
+        var emailSearchString = '?' + Querystring.stringify({ email: queryParams.email });
+        var endpoint = app + 'api/force_auth' + emailSearchString;
+        return this.parent
+          .then(openPage(endpoint, expectedHeader))
+          .then(function () {
+            if (Object.keys(queryParams).length) {
```

Comment:
Before the change, this test specified `> 1`, presumably because the email query param already got appended to `endpoint`. Is it intentional for that to have changed here?